src/Commands/Connection/InfoCommand.php

```diff
@@ -57,16 +61,9 @@ class InfoCommand extends TerminusCommand
      *   Display all of the connection information fields related to git.
      *
      */
-    public function connectionInfo($environment)
+    public function connectionInfo($site_env_id)
     {
-        $site_env = explode('.', $environment);
-        if (count($site_env) != 2) {
-            throw new \Exception('The environment argument must be given as <site_name>.<environment>');
-        }
-
-        $sites = new Sites();
-        $site  = $sites->get($site_env[0]);
-        $env   = $site->environments->get($site_env[1]);
+        list(, $env) = $this->getSiteEnv($site_env_id);
 
         return new AssociativeList($env->connectionInfo());
```

Comment:
While we don't need to do it in this PR, we should also think about how to inject formatters to so that it's easier to spy on them to unit test that they are called with appropriate parameters.